tests/partners/test_integration.py

```diff
@@ -34,21 +34,6 @@ def test_partner_is_returned(self):
         self.assertDictContainsDict(self.result, expected_result)
 
 
-class TestCreatePartnerDuplicateName(PartnerTestCase):
```

Comment:
Right, they changed it because partner's name isn't unique on our side.  Sorry, forgot about that.